api/src/main/java/io/opencensus/trace/internal/BaseMessageEventUtil.java

```diff
@@ -14,39 +14,46 @@
  * limitations under the License.
  */
 
-package io.opencensus.internal;
+package io.opencensus.trace.internal;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import io.opencensus.trace.MessageEvent;
+import io.opencensus.common.Internal;
 
 /**
- * Helper class to convert/cast between for {@link MessageEvent} and {@link NetworkEvent}.
+ * Helper class to convert/cast between for {@link io.opencensus.trace.MessageEvent} and {@link
+ * io.opencensus.trace.NetworkEvent}.
  *
  * @since 0.12
```

Comment:
I think the since tag should be removed, or updated to 0.13 for the new location.

Reply:
I just removed it since it's only for internal usage.